File: Assignment No.37/Program5.java
/*Problem Statement :Write a Program which accept one number and whether that 1,32 bit is on or off.
Input : 10 
Output : off
*/
import java.util.*;

class Program5
{
	public static boolean ChkBit(long iNo)
	{
		long iMask=0x80000001,iResult=0L;

		iResult=iNo | iMask;
		if(iResult == iMask)
		{
			return true;
		}
		else
		{
			return false;
		}
			 
	}
	public static void main(String args[])
	{
		long iNo = 0l;
		Boolean bret=false;

		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the Number :");
		iNo = sobj.nextLong();
		
		bret = ChkBit(iNo);
		if(bret == true)
		{
			System.out.println("1,32 bit is on");
		}
		else
		{
			System.out.println("1,32 bit is off");
		}
		
	}
}	
		

File: Assignment No 3/Program5.c
//Write a program which accept one character from user and check whether that character is vowel or not.
//Input : d
//Output : FALSE
#include<stdio.h> 
#include<stdbool.h>
#define true 1
#define false 0
bool DisplayVowel(char cValue)
{
	if(((cValue == 'A')||(cValue == 'a'))||(cValue == 'E')||(cValue == 'e')||(cValue == 'I')||(cValue == 'i')||(cValue == 'O')||(cValue == 'o')||(cValue == 'U')||(cValue == 'u'))
	{
		return true;
	}
	else 
	{
		return false;
	}
}
int main()
{
	char cValue='\0';
	bool bRet=false;
	printf("Enter character\n");

	scanf("%c",&cValue);

	bRet=DisplayVowel(cValue);

	if(bRet==true)
	{
		printf("It is a vowel\n");
	}
	else 
	{
		printf("It is not a vowel\n");
	}
	return 0;
}